frontend/src/views/stock/StockTableSFC.vue

```diff
@@ -19,89 +19,185 @@
  SPDX-License-Identifier: Apache-2.0
 -->
 <template>
-  <div class="flex flex-row">
-    <!--class="grid grid-rows-1 grid-flow-col gap-4"> -->
-    <div class="basis-1/2 pr-5">
-      <h2 class="text-center bold text-3xl">{{ title }}</h2>
-      <table class="">
-        <tr class="text-left">
-          <th>Material (ID)</th>
-          <th>Quantity</th>
-          <th>Allocated to Partner</th>
-          <th>Is Blocked</th>
-          <th>BPNS</th>
-          <th>BPNA</th>
-        </tr>
-        <tr
-            v-for="stock in stocks"
-            :key="stock.uuid"
-            @click="selectStock(stock.material.materialNumberCustomer, stock.uuid)"
-            :class="{ highlight: stock.material.materialNumberCustomer === selectedMaterialId }"
-        >
-          <td>{{ stock.material.name }}<br>({{ stock.material.materialNumberCustomer }})</td>
-          <td>{{ stock.quantity }} {{ getUomValueForUomKey(stock.measurementUnit) }}</td>
-          <td v-if="stock.type == 'PRODUCT'">{{ stock.allocatedToPartner.name }}<br>({{ stock.allocatedToPartner.bpnl }})</td>
-          <td v-if="stock.type == 'MATERIAL'"></td>
-          <td>{{stock.isBlocked}}</td>
-          <td>{{stock.stockLocationBpns}}</td>
-          <td>{{stock.stockLocationBpna}}</td>
-        </tr>
-      </table>
-    </div>
-    <div class="basis-1/2">
-      <PartnerStockSFC
-          :selectedMaterialOrProductId="this.selectedMaterialId"
-          :partnerRole="this.partnerRole"
-          :key="this.selectedMaterialId"
-      />
-      <!--
+    <div class="flex flex-col">
+        <div class="">
+            <h2 class="text-center bold text-3xl">{{ title }}</h2>
+            <h3 class="bold text-2xl">Your Stocks</h3>
+            <p v-if="this.ownRole === 'customer'">
+                <i
+                    >Info: These are your material stocks (your inputs) at your
+                    site.
+                    <b
+                        >Please select one of the material stocks to see the
+                        stocks your supplier still got on stock.</b
+                    ></i
+                >
```

Comment:
Please check the format in different parts so that HTML elements like `<i>` aren't split between lines.